spec/takeaway_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'takeaway_1'
```

Comment:
This seems strange to me - you are requiring  `takeaway_1` but your TakeAway class is in the file `takeaway`